annotation-processor/src/test/java/org/hibernate/validator/ap/ConstraintValidationProcessorTest.java

```diff
@@ -512,8 +516,8 @@ public void testThatNonUniqueValidatorResolutionCausesCompilationError() {
 		assertFalse( compilationResult );
 		assertThatDiagnosticsMatch(
 				diagnostics, new DiagnosticExpectation(
-				Kind.ERROR, 22
-		)
+						Kind.ERROR, 22
+				)
```

Comment:
I think the `new DiagnosticExpectation( Kind.ERROR, 22 )` should be on its own line, that would avoid formatting changes.

Better include it in this formatting PR I'm talking about in a previous comment.